app/src/main/res/values/arrays.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string-array name="java_version_entries">
+        <item>8</item>
+        <item>11</item>
```

Comment:
what about other version? For ex if you just want to use text blocks and want compatibility with Java 16, you can just use it

Reply:
okay, i will add every java version from 8 to 17.